common/transport/amqp/devdoc/amqp_message.md

```diff
@@ -40,31 +40,31 @@ As this is an internal API, the input argument message can be assumed to be of t
 
 **SRS_NODE_IOTHUB_AMQPMSG_05_001: [** The `fromMessage` method shall create a new instance of `AmqpMessage`. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_05_002: [** The created `AmqpMessage` object shall have a property of type `Object` named `properties`. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_05_002: [** The created `AmqpMessage` object shall NOT have a property of type `Object` named `properties`. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_05_003: [** If the `message` argument has a `to` property, the `properties` property of the `AmqpMessage` object shall have a property named `to` with the same value. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_05_003: [** If the `message` argument has a `to` property, the `AmqpMessage` object shall have a property named `to` with the same value. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_05_004: [** If the `message` argument has an `expiryTimeUtc` property, the `properties` property of the `AmqpMessage` object shall have a property named `absoluteExpiryTime` with the same value. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_05_004: [** If the `message` argument has an `expiryTimeUtc` property, the `AmqpMessage` object shall have a property named `absolute_expiry_time` with the same value. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_05_007: [** If the `message` argument has a `messageId` property, the `properties` property of the `AmqpMessage` object shall have a property named `messageId` with the same value. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_05_007: [** If the `message` argument has a `messageId` property, the `AmqpMessage` object shall have a property named `message_id` with the same value. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_011: [** If the `Message.messageId` property is a UUID, the AMQP type of the `AmqpMessage.properties.messageId` property shall be forced to UUID. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_011: [** If the `Message.messageId` property is a UUID, the AMQP type of the `AmqpMessage.message_id` property shall be forced to Buffer[16]. **]**
```

Comment:
>shall be forced to Buffer[16] [](start = 140, length = 29)

how is that realized? is that a rhea API call? that we'd need to mock?

Reply:
rhea.string_to_uud.

Seems un-necessary to mock it.

---
In reply to: [191905143](https://github.com/Azure/azure-iot-sdk-node/pull/300#discussion_r191905143) [](ancestors = 191905143)